src/app/components/footer/footer.component.ts

```diff
@@ -0,0 +1,17 @@
+import {ChangeDetectionStrategy, Component} from '@angular/core';
+
+@Component({
+  selector: 'app-footer',
+  templateUrl: './footer.component.html',
+  styleUrls: ['./footer.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush
+})
+
+export class FooterComponent {
+
+  date = new Date();
+  year = [this.date.getFullYear()]
```

Comment:
Maybe we can introduce convention to always add access modifier to property, that way it will be easier to notice the purpose of the property, so in that case for example I would go with simplified `public year: number = new Date().getFullYear();`
I'm also not sure why did you put this inside `[ ]`